src/test/java/step1/set/SetTest.java

```diff
@@ -0,0 +1,62 @@
+package step1.set;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("size 메소드로 Set의 크기를 확인할 수 있다.")
+    @Test
+    void size() {
+
+        // given & when
+        int result = numbers.size();
+
+        // then
+        assertThat(result).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(result).hasSize(3);
```
컬렉션을 쉽게 테스트할 수 있도록 메서드를 제공하고 있으니 확인해 보세요 😄

Reply:
hasSize 메소드 내부에서 IterableUtil.sizeOf에서 Collection이면 size 메소드를 사용해서 반환해주네요!
피드백 확인했습니다 감사합니다!